core/src/main/java/com/dtolabs/rundeck/core/jobs/JobEventStatus.java

```diff
@@ -0,0 +1,30 @@
+package com.dtolabs.rundeck.core.jobs;
+
+public class JobEventStatus {
```

Comment:
should be an interface, and should have a way to return a new `StepExecutionContext`.